### PR TITLE
Update dependencies on README.md

### PR DESCRIPTION
**Pull request by ben-grande**
Fixes https://github.com/QubesOS/qubes-issues/issues/8624

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ Local, and Qubes executors are available.
...
 ```bash
-$ sudo dnf install python3-packaging createrepo_c devscripts gpg python3-pyyaml rpm docker python3-docker podman python3-podman reprepro python3-pathspec rpm-sign mktorrent openssl tree mock python3-jinja2-cli pacman m4 asciidoc rsync
+$ sudo dnf install python3-packaging python3-click python3-lxml createrepo_c devscripts gpg python3-pyyaml rpm docker python3-docker podman python3-podman reprepro python3-pathspec rpm-sign mktorrent openssl tree mock python3-jinja2-cli pacman m4 asciidoc rsync
 ```
 
 Debian:
 
 ```bash
-$ sudo apt install python3-packaging createrepo-c devscripts gpg python3-yaml rpm docker.io python3-docker reprepro python3-pathspec mktorrent openssl tree python3-setuptools python3-lxml python3-dateutil
+$ sudo apt install python3-packaging python3-click python3-lxml createrepo-c devscripts gpg python3-yaml rpm docker.io python3-docker reprepro python3-pathspec mktorrent openssl tree python3-setuptools python3-lxml python3-dateutil
```
Comment (Augsch123) [inline README.md:26]: `python3-lxml` has appeared in the latter part of this line.

**Review by Augsch123 — COMMENTED** [review]
Thanks for doing this! Could you add another note in the `Qubes executer` part that `rpc/qubesbuilder.FileCopyIn` and `rpc/qubesbuilder.FileCopyOut` must be made executable through `chmod`?

File: README.md (hunk 1 above)
Comment (ben-grande) [reply README.md]: Fixed

**Comment by ben-grande** [comment]
> Thanks for doing this! Could you add another note in the `Qubes executer` part that `rpc/qubesbuilder.FileCopyIn` and `rpc/qubesbuilder.FileCopyOut` must be made executable through `chmod`?

Unrelated changes to this PR.
Opened
- https://github.com/QubesOS/qubes-builderv2/pull/74.

**Review by marmarek — APPROVED** [review]

**Pull request merged**